2020-03-Autumn/3_backtracking/README.md

```diff
@@ -58,21 +58,37 @@ Go to [Solution](#s1)   [Top](#top)
 <!-- Don't remove -->
 <a name="p2"/>
 
-### 2. PROBLEM 2 TODO :bug:
+### 2. PROBLEM 2 Subsets
 
-Source: TODO :bug:
+Source: [LeetCode](https://leetcode.com/problems/subsets/)
 
 #### Scenario
 
-Problem Statement TODO :bug:
+Given an integer array nums, return all possible subsets (the power set).
 
-#### Example Input
+The solution set must not contain duplicate subsets.
 
-If the problem is simple enough, remove this section. TODO :bug:
+#### Example
+
+Example 1:
+
+Input: nums = [1,2,3]
+Output: [[],[1],[2],[1,2],[3],[1,3],[2,3],[1,2,3]]
+Example 2:
+
+Input: nums = [0]
+Output: [[],[0]]
+
+#### Constraints
+
+1 <= nums.length <= 10
+-10 <= nums[i] <= 10
+All the numbers of nums are unique.
 
 #### Function Signature
 
-TODO :bug:
+C++: vector<vector<int>> subsets(vector<int>& nums)
+Java: public List<List<Integer>> subsets(int[] nums)
```

Comment:
pls use inline code or code block for it

Reply:
How do I change it to code block?

Reply:
use \' \' for inline code
use \``` \``` for code block

https://guides.github.com/features/mastering-markdown/#syntax